onnxruntime/core/session/inference_session.cc

```diff
@@ -1377,12 +1394,12 @@ common::Status InferenceSession::Initialize() {
       }
 
 #if defined(ORT_ENABLE_ORT_FORMAT_RUNTIME_GRAPH_OPTIMIZATION)
-      ORT_RETURN_IF_ERROR_SESSIONID_(TransformGraphForOrtFormatModel(graph, *session_logger_));
+      ORT_RETURN_IF_ERROR_SESSIONID_(ReplayRuntimeOptimizations(graph, *session_logger_, session_options_));
```

Comment:
>ReplayRuntimeOptimizations [](http://example.com/codeflow?start=37&length=26)

nit: would ReplaySavedOptimizations be clearer?

Reply:
renamed to ReplaySavedRuntimeOptimizations